loom/graphics/CMakeLists.txt

```diff
@@ -60,6 +57,8 @@ if (FLASH)
 
 endif()
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x" )
```

Comment:
https://github.com/LoomSDK/LoomSDK/blob/JG-ShaderObjects/CMakeLists.txt#L75

I have already fixed this in my shader objects branch. Also, only clang/gcc accept this parameter. Msvc generates a warning, it's enabled by default anyway.